docs/introduction/overview.rst

```diff
@@ -2,27 +2,29 @@
 Before you start
 ################
 
-| This tutorial offers Java developers all the needed information to build a complete blockchain application on the Horizen Sidechain system.
+| This tutorial offers Java developers all the information needed to build a complete blockchain application on the Horizen Sidechain system.
 
 | Apart from Java competency, this tutorial assumes that the reader has a high-level understanding of how blockchain-based distributed software works.
 
-So, concepts such as Transactions, UTXO's, Blocks, Validation, Confirmation, Consensus, Unique chain, and chain forks, Hash Function, Private/Public key, and Signing should be known and understood, as well as the concept of a network of nodes and node communication.
+So, you should be comfortable with concepts such as Transactions, UTXO's, Blocks, Validation, Confirmation, Consensus, Unique Chain, Chain Forks, Hash Function, Private/Public key, and Signing, along with the concept of a network of nodes and node communication.
```

Comment:
@zennermac 

You should be comfortable with concepts like transactions, UTXO's, blocks, validation, confirmation, consensus, unique chains, chain forks, hash functions, private/public key,s and signing, along with the concept of a network of nodes and node communication.